test/from-draft2/to-draft3/requires-is-present-in-properties-children.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "title": "Properties has children who have requires and dependencies is present in outer schema",
+        "from": {
+            "properties": {
+                "foo": {
+                    "requires": "bar"
+                },
+                "bar": {
+                    "requires": { "type": "string"}
+                }
+            },
+            "dependencies": {}
+        },
+        "to": {
+            "properties": {
+                "foo": {},
```

Comment:
The test is definitely valid, but I wonder if we should get rid of these empty properties. No reason to artificially add them, right?

Reply:
Okay, i'll have to add rules for whose dependent rule is popping properties..

Reply:
Right. OK, fine to keep them then!